05/app.js

```diff
@@ -10,23 +10,24 @@ const stats = {
 /* tutaj umieść swój kod */
 const pElementsList = document.querySelectorAll('.text');
 
+const countClicks = function (firstKey, secondKey) {
```

Comment:
Super, o to chodziło!
Jeszcze doczepiłbym się do nazw parametrów :P
Sam bym użył np. `elementName` oraz `elementId` - aby lepiej wskazać czym są :)